examples/models/mlflow_v2_protocol_end_to_end/mlflow_v2_protocol_end_to_end.ipynb

```diff
@@ -0,0 +1,902 @@
+{
```

Comment:
<p>Would be good to add subtitels to these sections as well</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/3483/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>